src/Codec/Xlsx/Types/DataValidation.hs

```diff
@@ -183,20 +217,39 @@ readValidationType op ty cur = do
     opExp <- readOpExpression2 op cur
     readValidationTypeOpExp ty opExp
 
-readListFormulas :: Formula -> Maybe [Text]
-readListFormulas (Formula f) = readQuotedList f
+-- | Attempt to obtain a plain list expression
+maybePlainValidationList :: ValidationType -> Maybe ValidationList
+maybePlainValidationList (ValidationTypeList (ListExpression le)) = Just le
+maybePlainValidationList _ = Nothing
+
+-- | Attempt to obtain a range expression
+maybeRangeValidationList :: ValidationType -> Maybe Range
```

Comment:
why do we still have "List" in this name when it returns (optional) range, i.e. single one with no lists whatsoever?

Reply:
The reasoning it that it unpacks (tries to) a `ValidationTypeList` (i.e. `<dataValidation type="list"> )

The range is just a reference to list (contents of referenced cells)

If you really do not like it I'll rename it :)

Reply:
I think this "List" part is a part of inner detail which doesn't make sense at least from the signature side which makes it slightly confusing, what do you think about just dropping it and leaving it as just `mabyeRangeValidation` or maybe `maybeValidationRange`?